api/namex/constants/__init__.py

```diff
@@ -380,6 +394,9 @@ class LegacyEntityTypes:
     ('CCC', EntityTypes.COMMUNITY_CONTRIBUTION_COMPANY.value, RequestAction.RESUBMIT.value),
     ('CCCT', EntityTypes.COMMUNITY_CONTRIBUTION_COMPANY.value, RequestAction.MVE.value, True),
     ('CCCT', EntityTypes.COMMUNITY_CONTRIBUTION_COMPANY.value, RequestAction.RESUBMIT.value),
+    ('CCCT', EntityTypes.CONTINUE_IN_CCC.value, RequestAction.RESUBMIT.value),
+    ('CCCT', EntityTypes.CONTINUE_IN_CCC.value, RequestAction.REH.value),
+    ('CCCT', EntityTypes.CONTINUE_IN_CCC.value, RequestAction.CHG.value),
```

Comment:
Same comment as above. Did you find legacy code CCCT for CHG?

If not then I think you should change CHG to CCC (and add an entry for RESUBMIT).

The code the REH (and also REN and REST) should be RCC, as below, I think.

Reply:
Before we go too far here, can you please provide a list of legacy codes for continued in BC Limited, CCC and ULC?

Reply:
Is CCC an entity type? the corresponding request type is 'CCCT'. Could you give me a sample in DEV?

> Same comment as above. Did you find legacy code CCCT for CHG?
> 
> If not then I think you should change CHG to CCC (and add an entry for RESUBMIT).
> 
> The code the REH (and also REN and REST) should be RCC, as below, I think.

Is CCC an entity type? the corresponding request type is 'CCCT'. Could you give me a sample in DEV?

Reply:
> Before we go too far here, can you please provide a list of legacy codes for continued in BC Limited, CCC and ULC?

Sorry, I don't have legacy code.

Reply:
> > Before we go too far here, can you please provide a list of legacy codes for continued in BC Limited, CCC and ULC?
> 
> Sorry, I don't have legacy code.

Whatever you did for these CHG codes, please do for REH/REN/REST.

![image](https://github.com/user-attachments/assets/1ea3b3d0-eb5d-4d9b-9b21-fa78961b03f8)

Reply:
I don't believe there are mappings for 'continue-in' from REN or REH in namerequest, as seen here: https://github.com/bdgov/namerequest/blob/main/app/src/list-data/request-action-mapping.ts#L82. Is there a specific reason to include this code in namex?

Reply:
That mapping appears to be for the drop-down menu in Namex:

![image](https://github.com/user-attachments/assets/bbc0e316-cac7-4eab-9201-dadad92b802c)

I think you are right: there are no special request type codes for the continued in businesses because continued in businesses use the existing regular request types (eg, `CT` is used for both Corporation and ContinueIn).

However, I think it's worth a look at this mapping to ensure that all possible conversions exist. I finally have the requirements. Here are all possible conversions:
1. BC Limited -> Benefit Company
2. BC Limited -> Unlimited Liability Company
3. BC Limited -> Community Contribution Company
4. Benefit Company -> BC Limited
5. Benefit Company -> Community Contribution Company
6. Unlimited Liability Company -> BC Limited
7. Unlimited Liability Company -> Benefit Company

Can you please verify that we have request type codes for these 7 conversions, and the required entries in both mapping tables? Thanks!

Reply:
> 5. Benefit Company -> Community Contribution Company

I believe we are missing a request type code for this. Can we just make one up? How about `BECC`?

Reply:
Resolving for now. Will review new code changes instead.